buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -85,7 +85,7 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "exec $(PB_DockerContainerName) $(PB_GitDirectory)/build.sh $(BuildArguments) $(portableLinux) --env-vars \"DISABLE_CROSSGEN=1,TARGETPLATFORM=$(PB_Architecture),TARGETRID=$(TargetRid),PUBLISH_TO_AZURE_BLOB=$(PUBLISH_TO_AZURE_BLOB)\"",
+        "arguments": "exec -e PUBLISH_TO_AZURE_BLOB=$(PUBLISH_TO_AZURE_BLOB) -e CONNECTION_STRING=$(CONNECTION_STRING) -e CLI_NUGET_FEED_URL=$(CLI_NUGET_FEED_URL) -e CLI_NUGET_API_KEY=$(CLI_NUGET_API_KEY) -e NUGET_FEED_URL=$(NUGET_FEED_URL) -e NUGET_API_KEY=$(NUGET_API_KEY) -e GITHUB_PASSWORD=$(GITHUB_PASSWORD) $(PB_DockerContainerName) $(PB_GitDirectory)/build.sh $(BuildArguments) $(portableLinux) --env-vars \"DISABLE_CROSSGEN=1,TARGETPLATFORM=$(PB_Architecture),TARGETRID=$(TargetRid)\"",
```

Comment:
See how we do it in the CLI:

https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dockerrun.sh#L115-L136

```
docker run $INTERACTIVE -t --rm --sig-proxy=true \
    --name $DOTNET_BUILD_CONTAINER_NAME \
    -v $DOCKER_HOST_SHARE_DIR:/opt/code \
    -e CHANNEL \
    -e CONNECTION_STRING \
    -e REPO_ID \
    -e REPO_USER \
    -e REPO_PASS \
    -e REPO_SERVER \
    -e DOTNET_BUILD_SKIP_CROSSGEN \
    -e PUBLISH_TO_AZURE_BLOB \
    -e NUGET_FEED_URL \
    -e NUGET_API_KEY \
    -e GITHUB_PASSWORD \
    -e ARTIFACT_STORAGE_KEY \
    -e ARTIFACT_STORAGE_ACCOUNT \
    -e ARTIFACT_STORAGE_CONTAINER \
    -e CHECKSUM_STORAGE_KEY \
    -e CHECKSUM_STORAGE_ACCOUNT \
    -e CHECKSUM_STORAGE_CONTAINER \
    -e CLIBUILD_SKIP_TESTS \
    -e CommitCount \
```

It makes it less verbose.